releasepolicy.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: contribute
+title: Release Policy
+---
+
+# Release policy
+
+We regularly publish [binary releases of Bazel](https://github.com/bazelbuild/bazel/releases).
+
+A release candidate for a new MINOR version of Bazel is usually created at the beginning of
+every month. The work is tracked by a
+[release bug on GitHub](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+is%3Aopen+label%3Arelease)
+indicating a target release date, and is assigned to the current Release manager.
+Release candidates should pass all our unit tests, and show no unwanted regression in the
+projects tested on [Buildkite](https://buildkite.com/bazel).
+
+Release candidates are announced on [bazel-discuss](https://groups.google.com/forum/#!forum/bazel-discuss).
+Over the next days, the Bazel team monitors community bug reports for any regressions in the candidates.
+
+If no regressions are discovered, the candidate is officially released after one week. However,
+regressions can delay the release of a release candidate. If regressions are found, the Bazel team applies
+corresponding cherry-picks to the release candidate to fix those regressions. If no further
+regressions are found for two business days, but not before one week has elapsed since the first
+release candidate, the candidate is released.
+
+New features are not cherry-picked into a release candidate after it is cut.
+Moreover, if a new feature is buggy, the feature may be rolled back from a
+release candidate. Only critical, high-impact bugs will be fixed in a release candidate.
+
+A release is only released on a day where the next day is a business day.
+
+If a critical issue is found on the latest release, a patch release can be emitted by applying the
```

Comment:
I don't really understand this paragraph. Can we rephrase?

If a critical issue is found in the latest release, the Bazel team creates a _patch release_ by applying the fix to the release. Because this patch updates an existing release instead of creating a new one, the patch release candidate can be released after two business days.

Reply:
Done.